README.md

```diff
@@ -29,7 +30,10 @@ With `--ethereum` being the default option.
 
 When passing `--endpoint` or `-e`, the second argument must be the `http` endpoint
 of the node you are connecting to. If choosing from a preset chain and specifying
-a test, `--` should be passed for the `<URL>` argument.
+a test, `--` should be passed for the `<URL>` argument. You must also set the
+`USE_REVIVE` env var to either `true` or `false`, in order to enable compilation
```

Comment:
if a user chooses Westend as network and `false`, what does that mean? will that give any meaningful output?

Reply:
Got to update that bit. If you choose any of the preset chains, the script sets the value of `USE_REVIVE`, so it shouldn't matter. If on the other hand you are using `-e`, then you should set the value of `USE_REVIVE` depending on the chain you are connecting to.

Reply:
Could you check again now?